fastlane/lib/fastlane/fast_file.rb

```diff
@@ -24,6 +24,14 @@ def initialize(path = nil)
                 'you should turn off smart quotes in your editor of choice.'
       end
 
+      content.scan(/(.*require) (.*)/).each do |current|
+        require_command = current.first
+        gem_name = current.last
+        next if require_command.include?("fastlane_require")
```

Comment:
Use a regex that only matches require and not `fastlane_require`, like `^\s*(require) (.*)`